all_msgs/ros2interop/ros-types-all.xml

```diff
@@ -0,0 +1,1445 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+(c) 2019 Copyright, Real-Time Innovations, Inc.  All rights reserved.
```

Comment:
Change the license here with the same license we use in the IDL files in this repository.